android/src/main/kotlin/com/builttoroam/devicecalendar/common/Constants.kt

```diff
@@ -62,7 +63,9 @@ class Constants {
                 CalendarContract.Events.EVENT_LOCATION,
                 CalendarContract.Events.CUSTOM_APP_URI,
                 CalendarContract.Events.EVENT_TIMEZONE,
-                CalendarContract.Events.EVENT_END_TIMEZONE
+                CalendarContract.Events.EVENT_END_TIMEZONE,
+                CalendarContract.Events.AVAILABILITY,
+                CalendarContract.Events.STATUS
```

Comment:
@laura-digio I noticed that you added .STATUS here - is this needed in order to get availability information? or was there another bug that this is fixing?

Reply:
@nickrandolph sorry, it's not necessary. I fix this change!
Sorry and thanks for look the PR